app/modules/events.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var formatMessage = function(message)
```

Comment:
why in a variable and not just naming the function? Don't stack traces come out with anonymous functions that way..?